cql/DashboardLibrary.json

```diff
@@ -644,6 +644,14 @@
             "codeSystem" : {
                "name" : "SNOMED-CT"
             }
+         }, {
```

Comment:
When updating a CQL file that has it's own [FHIR library](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/tree/main/fsh-tank/fsh-generated/resources), you will need to run our packager script (`npm run packager`). This script will automatically compile ELM into a base64 format and copy it to the appropriate inline places in the FHIR Shorthand and FHIR Library files.

I usually run the prepack script (`npm run prepack`), which will:
-  run `npx cql-to-elm cql`
-  Run the packager script
- Copy the build into the `dist/` folder

Reply:
Got it. Will do this as a last step before merge.

Reply:
I ran the packager script to generate update FSH files: https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/154/commits/65459cfe2066495ec09cdb8cd72e8be52ba63cb2.